frontend-sveltekit/package.json

```diff
@@ -16,35 +16,35 @@
     },
     "devDependencies": {
         "@playwright/test": "^1.28.1",
-        "@sveltejs/adapter-auto": "^2.0.0",
+        "@skeletonlabs/skeleton": "2.1.0",
+        "@skeletonlabs/tw-plugin": "0.2.0",
+        "@sveltejs/adapter-node": "^1.3.1",
```

Comment:
`adapter-node` installed and `adapter-auto` removed

all the rest is whitespace changes from the node command line